### DISCUSSION TITLE
Customized IBM Notifier Notification Alert showing Circle / Slash Icon

### DISCUSSION THREAD

**Discussion by marcusheBG** [IBM/mac-ibm-notifications#140] [Q&A] [answered]
Hello!

Thank you in advance. We've noticed our customized IBM Notifier since Ventura is showing notification alerts with a circle / slash / crossed out / no over the icon.

However the app is signed, and we have a PPPC notification profile applied.

The customized app does show a correct icon for the pop-up type & in the Finder for the Application itself. A type of banner as opposed to a type of alert does show the correct icon without the circle / slash. The vanilla IBM Notifier app does not exhibit this issue when showing an alert - only our customized app.

The functionality is perfect otherwise - but we do not understand why we are seeing the circle / slash icon for alerts.

Could it be our signature, app permissions, Ventura bug, Apple Silicon ARCH build-time bug, or something else?

We found a discussion of a similar issue a user was having with multiple apps showing a circle / slash here:

https://discussions.apple.com/thread/253503524

Seeming like notification permissions at the moment... will try some things with the PPPC

- Marcus

<img width="370" alt="Screenshot 2022-11-14 at 10 00 17 AM" src="https://user-images.githubusercontent.com/90919338/201707129-f9e8aebe-233a-4fcd-aa76-c18e89e72ac3.png">

<img width="370" alt="Screenshot 2022-11-14 at 3 32 57 PM" src="https://user-images.githubusercontent.com/90919338/201760681-32b7097e-3341-48dd-b39f-0abaed558cc3.png">

<img width="412" alt="Screenshot 2022-11-14 at 11 11 11 AM" src="https://user-images.githubusercontent.com/90919338/201709119-039db9b6-ced9-44e5-a5c5-288768605d17.png">

**Comment by marcusheBG** [comment, marked as answer]
Hello,

Apologies to answer our own question.

The issue ended up being a partially corrupted notifications profile in our MDM (Mosyle)

We had to make a new notifications profile and upload to the MDM. Our previous profile was downloaded from Jamf and reuploaded to Mosyle. This time we made a new profile from scratch using iMazing Profile Editor.

FYI for anyone finding this in the future for their circle / slash notification icon issue.

Thanks!

**Comment by SMartorelli** [comment]
Hi @marcusheBG, thanks for finding a solution to this. It's actually something that multiple users reported but we didn't experienced it specifically so it's good to have new information about a possible resolution. Going to mark your second message as response for this thread.

**Reply by marcusheBG** [reply]
Thanks @SMartorelli !

Just reporting in - our custom builds are still sometimes experiencing this issue even with a known good notifications profile. We did have a corrupted profile before - but it seems to be some combination of reapplying the profile, rebooting, or perhaps needing to formally "open" the notifier .app one time without a command passed to sort of "register it" (could be a post install command of `"open /path/to/notifierapp"`

Feels to me this is more of a macOS Services or Swift issue that something with IBM Notifier, as other Apple proprietary apps are able to experience it as well.

![image](https://user-images.githubusercontent.com/90919338/205695848-c64d7ef4-b1f5-47d0-9f60-d676b6cb75aa.png)
